packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -592,7 +606,7 @@ export abstract class SharedObjectCore<
 	 * @returns `true` if the event had listeners, `false` otherwise.
 	 */
 	public emit(event: EventEmitterEventType, ...args: any[]): boolean {
-		return this.callbacksHelper.measure(() => {
+		return this.callbacksHelper.measure<boolean>(() => {
```

Comment:
Initially expected compiler to fail when passing `boolean` (as `K keyof TKey`: `K extends string ? number : never`), but somehow is accepting this type parameter.